src/Module/NodeInfo.php

```diff
@@ -31,8 +22,12 @@ public static function rawContent(array $parameters = [])
 		if (($app->argc > 1) && ($app->argv[0] == '.well-known')) {
 			self::printWellKnown($app);
```

Comment:
Please move this in a new `Module\WellKnown\NodeInfo` module since the route has changed.

Reply:
The route stayed the same, I just added another nodeinfo version.

Reply:
It used to be `/.well-known/nodeinfo[/1.0]` and `/nodeinfo/1.0`, now it's `/.well-known/nodeinfo` on one hand, and `/nodeinfo/{version}` on the other hand. Both are displaying very different information for two different routes and shouldn't be clumped in the same module.

`/.well-known/nodeinfo` should use `Module/WellKnown/NodeInfo` (to be created)
and
`/nodeinfo/{version}` should use `Module\Nodeinfo` (the current one)

Reply:
It used to be `/.well-known/nodeinfo[/1.0]` and `/nodeinfo/1.0`, now it's `/.well-known/nodeinfo` on one hand, and `/nodeinfo/{version}` on the other hand. Both are displaying very different information for two different routes and shouldn't be clumped in the same module.

`/.well-known/nodeinfo` should use `Module/WellKnown/NodeInfo` (to be created)
and
`/nodeinfo/{version}` should use `Module\Nodeinfo` (the current one)

Reply:
The `[/1.0]` had been an error that I corrected. I will have a look at it the next days.

Reply:
I understand, I still think the module should be split. It probably should have been split before, but I didn't have an occasion to look at it that closely.

Reply:
`Cannot declare class Friendica\Module\NodeInfo, because the name is already in use in /var/www/virtual/pirati.ca/htdocs/src/Module/WellKnown/NodeInfo.php on line 0`